frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx

```diff
@@ -21,12 +23,18 @@ const setupPremium = (opts?: SetupOpts) => {
 
 describe("[EE, with token] embedding settings", () => {
   describe("when the embedding is disabled", () => {
-    describe("static embedding", () => {
-      it("should not allow going to static embedding settings page", async () => {
-        const { history } = await setupPremium({
+    let history: Awaited<ReturnType<typeof setupEmbedding>>["history"];
+
+    beforeEach(async () => {
+      history = (
+        await setupPremium({
```

Comment:
Just curious, what is the difference between Premium and Enterprise tests? They seem to be duplicated at first glance.

Reply:
[answered here.](https://github.com/metabase/metabase/pull/47519#discussion_r1743458266)